{{cookiecutter.github_repository}}/settings/__init__.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Are we getting rid of these?

Long time 😢

Reply:
yeah. sadly. Python3 is unicode by default.

Reply:
Not too sad if its default 👌